README.md

```diff
@@ -34,6 +34,9 @@ or an IObservable based GitHub API client library for .NET using Reactive Extens
 ```
 Install-Package Octokit.Reactive
 ```
+### Beta packages ###
+The current master repo is availabe as nuget package at 
```

Comment:
Unstable NuGet packages that track the `master` branch of this repository are available at